frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -0,0 +1,300 @@
+import './BuildingForm.scss';
```

Comment:
Up to you if you want to change this but:

Do you think that SidebarContents is redundant? IE. are there any components that would be part of mapSideBar but not SidebarContents?